superset/viz.py

```diff
@@ -177,6 +177,11 @@ def process_metrics(self) -> None:
         self.all_metrics = list(self.metric_dict.values())
         self.metric_labels = list(self.metric_dict.keys())
 
+    def construct_where(self, query_obj: QueryObjectDict, predicate_string: str) -> str:
+        if query_obj:
+            predicate_string = "({}) AND ({})".format(query_obj, predicate_string)
```

Comment:
The signature here doesn't seem correct, `query_obj` here seems to refer to the original `where` and be `Optional[str]`?

Reply:
@villebro makes sense. Thanks for the catch!